src/lib/report_graphite.js

```diff
@@ -4,25 +4,24 @@ var _ = require('lodash');
 module.exports = function(crawlMetrics, graphiteClient) {
 
   var metrics = {
-    crawler: {
-      publicKeyCount: Object.keys(crawlMetrics.rippleds).length,
-      connectionsCount: Object.keys(crawlMetrics.connections).length,
-      rippleds: {}
+    crawlerTest: {
+        publicKeyCount: Object.keys(crawlMetrics.rippleds).length,
+        connectionsCount: Object.keys(crawlMetrics.connections).length,
+        rippleds: {}
     }
   };
   _.each(Object.keys(crawlMetrics.rippleds), function (rippled) {
-    metrics.crawler.rippleds[rippled] = {
+    metrics.crawlerTest.rippleds[rippled] = {
       connectionsCount: crawlMetrics.rippleds[rippled].in + crawlMetrics.rippleds[rippled].out,
       uptime: crawlMetrics.rippleds[rippled].uptime,
       up: 1
     };
   });
-
   graphiteClient.write(metrics, moment(crawlMetrics.crawl.start).valueOf(), function(error) {
     if (error) {
       throw new Error(error);
     } else {
-      console.log('Reported  crawl %d \t at %s \t to graphite',
+      console.log('Reported  crawl %s \t at %s \t to graphite',
         crawlMetrics.crawl.id, moment().format());
```

Comment:
crawlMetrics.crawl is in following format:
{
    'id': crawl_row.rowkey,
    'start': hbaseUtils.keyToStart(crawl_row.rowkey),
    'end': hbaseUtils.keyToEnd(crawl_row.rowkey),
    'entry': crawl_row.entry_ipp
  };